lib/views/help/ico_officers.html.erb

```diff
@@ -0,0 +1,128 @@
+      <a href="#handling_website_complaints">#</a>
+    </dt>
+    <dd>
+      <p>
+      We sometimes receive requests to remove part or all of a request or response. These are considered in accordance with our <a href="/help/how">takedown policy</a>. In most cases, an individual or authority who has referred our handling of data to the ICO or OSIC will have first made a complaint directly to us, and we will have considered it in line with these policies, balancing the impact on individuals’ rights, freedoms and interests against our legitimate interest in running the WhatDoTheyKnow service.
+      </p>
+      <p>
+      We are always happy to discuss such cases with you: please do <a href="/help/contact">get in touch</a>.
+    </p>
+    </dd>
+    <dt id="data_breach">
+    I’m handling a data protection breach that WhatDoTheyKnow reported to us
+      <a href="#data_breach">#</a>
+    </dt>
+    <dd>
+      <p>
+      With hundreds of requests processed by WhatDoTheyKnow every day, we don’t check the contents of every single one — but under our policy of <a href="/help/how">reactive moderation</a>, when we notice that an authority has accidentally released sensitive information or someone reports a serious data breach to us, we take it offline as soon as possible and report it to the ICO or OSIC where applicable.
+      </p>
+      <p>
+      In such cases, we are happy to help you understand what has happened and to provide any further details you require to help you handle the report. Note that we are neither the subject of the data breach nor the authority that released the sensitive data in such cases.
+    </p>
+    <p>
+      We try to help public bodies improve their processes so that accidental releases like this don’t happen, and, abiding by the ICO’s own guidance on when to do so, we report cases to you if applicable.
+    </p>
+
+    </dd>
+    <dt id="large_data_transfers">
+      I need you to transfer a large amount of personal data securely, for my investigations into a data breach
+      <a href="#large_data_transfers">#</a>
+    </dt>
+    <dd>
+      <p>
+      Where appropriate, we can provide data to aid with your investigations. Before we do so, we require details of how it will be used and why it is needed, which will be considered according to our policies.
+      </p>
+      <p>
+      We have procedures to ensure that such data is transferred securely. Please <a href="/help/contact">get in touch</a> with us to discuss details.
+      </p>
+    </dd>
+    <dt id="requests_over_time">
+      I am trying to understand whether an authority is meeting its duties under the law
+      <a href="#requests_over_time">#</a>
+    </dt>
+    <dd>
+      <p>
+      Because requests and responses are published openly on the site, WhatDoTheyKnow can be a useful tool for assessing how authorities are dealing with requests over time. Please do <a href="/help/contact">get in touch</a> if you would like to discuss this further.
```

Comment:
```suggestion
      Because requests and responses are published openly on the site, WhatDoTheyKnow can be a useful tool for assessing how authorities are dealing with requests over time. Please do <%= link_to 'get in touch', help_contact_path %> if you would like to discuss this further.
```